source/beamline/IDEAS/IDEASBeamline.h

```diff
@@ -218,8 +208,20 @@ protected slots:
 	CLSSIS3820Scaler *scaler_;
 
 	IDEASKETEKDetector *ketek_;
+	AMControl *ketekPeakingTime_;
```

Comment:
@dretrex Remember when we had that discussion about the "one-off" features? I had removed them from IDEASBeamline but you advised to put them back in. Those are for the exposedControls in IDEASBeamline::setupExposedControls.